testsuite/pytests/test_eprop_plasticity.py

```diff
@@ -551,49 +547,39 @@ def calculate_glorot_dist(fan_in, fan_out):
     weights_out_rec = np.array(np.random.randn(n_rec, n_out), dtype=dtype_weights)
 
     params_common_syn_eprop = {
-        "adam_beta1": 0.9,
-        "adam_beta2": 0.999,
-        "adam_epsilon": 1e-8,
-        "optimizer": "adam",
-        "batch_size": n_batch,
+        "optimizer": {
+            "type": "adam",
+            "batch_size": n_batch,
+            "eta": 5e-3,
+            "Wmin": -100.0,
+            "Wmax": 100.0,
+            "beta_1": 0.9,
+            "beta_2": 0.999,
```

Comment:
```suggestion
            "batch_size": n_batch,
            "beta_1": 0.9,
            "beta_2": 0.999,
            "eta": 5e-3,
            "Wmax": 100.0,
            "Wmin": -100.0,
```
The suggested change sorts the parameters after the `type` alphabetically.